docs/content/index.fsx

```diff
@@ -63,22 +54,29 @@ data
 |> Array.map (fun x -> x + 1L)
 |> Array.sum
 
+// Real: 00:00:00.044, CPU: 00:00:00.046, GC gen0: 0, gen1: 0, gen2: 0
+data
+|> Nessos.Streams.Stream.ofArray
+|> Nessos.Streams.Stream.filter (fun x -> x % 2L = 0L)
+|> Nessos.Streams.Stream.map (fun x -> x + 1L)
+|> Nessos.Streams.Stream.sum
```

Comment:
Because one can distinguish in a quicker way that it's using the library in this block.